usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -27,6 +27,15 @@ def preprocess(keyword):
     return keyword
 
 
+def es_sanitize(keyword):
+    """ Escapes reserved elasticsearch characters and removes when necessary """
+    processed_string = re.sub(r'([-&!|{}()^~*?:\\/"+\[\]<>])', '', keyword)
+    if len(processed_string) != len(keyword):
+        msg = "Stripped characters from ES city search string New: '{}' Original: '{}'"
```

Comment:
Is this going to be reserved just for ES city search?

Reply:
No, this has been cherry-picked over into the autocomplete as well. It will be used for all ES searches in time.

Reply:
Its not just for city search, I've fixed this in my other branch and will deal with the merge conflict so that its more clear.